consensus/event_loop.go

```diff
@@ -680,6 +681,16 @@ func (fe *fastHotStuffEventLoop) tryConstructVoteQCInCurrentView() *FastHotStuff
 	// Fetch the validator list at the tip.
 	validatorList := fe.tip.validatorList
 
+	for _, validator := range validatorList {
```

Comment:
I wrote a function called DomainsToString(). Let's use that here and elsewhere instead of iterating over GetDomains().